user/mods/ServerValueModifier/Presets/The Escape.json

```diff
@@ -497,24 +497,24 @@
     }
   },
   "CSM": {
-    "EnableCases": true,
+    "EnableCases": false,

```

Comment:
TODO: completely disable thid part (since we enabled `DefaultPocket` and our current profiles are saved now)